test/pet.test.js

```diff
@@ -15,37 +15,43 @@ describe('constructor', () => {
         expect(new Pet('Manny').name).toBe("Manny");
     })
 
-    it('should return a new pet with an age', () => {
+    it('should return a new pet with a default age of 0', () => {
         expect(new Pet().age).toBe(0);
```

Comment:
are you defining a default pet as testPet in your before each, if so , you can use it here as it will be reset after each test.